fe/fe-core/src/main/java/org/apache/doris/persist/EditLog.java

```diff
@@ -2102,4 +2102,17 @@ public String getNotReadyReason() {
         }
         return "";
     }
+
+    public boolean exceedMaxJournalSize(BackupJob job) {
+        try {
+            return exceedMaxJournalSize(OperationType.OP_BACKUP_JOB, job);
+        } catch (Exception e) {
+            LOG.warn("exceedMaxJournalSize exception:", e);
+        }
+        return true;
+    }
+
+    private boolean exceedMaxJournalSize(short op, Writable writable) throws IOException {
```

Comment:
This function is redundant if we do not care about `op`

Reply:
> This function is redundant if we do not care about `op`

opCode is important, we will print opCode